### PR TITLE
[Merged by Bors] - Node endpoints

### PR DESCRIPTION
**Pull request by realbigsean**
## Issue Addressed

`node` endpoints in #1434

## Proposed Changes

Implement these:
```
 /eth/v1/node/health
 /eth/v1/node/peers/{peer_id}
 /eth/v1/node/peers
```
- Add an `Option<Enr>` to `PeerInfo`
- Finish implementation of `/eth/v1/node/identity`

## Additional Info
- should update the `peers` endpoints when #1764 is resolved

### CODE REVIEW

File: beacon_node/eth2_libp2p/src/peer_manager/peer_info.rs [hunk 1]
```diff
@@ -46,6 +47,9 @@ pub struct PeerInfo<T: EthSpec> {
     /// Direction of the first connection of the last (or current) connected session with this peer.
     /// None if this peer was never connected.
     pub connection_direction: Option<ConnectionDirection>,
+    /// The enr of the peer, if known.
+    #[serde(skip)]
```
Comment (AgeManning) [inline beacon_node/eth2_libp2p/src/peer_manager/peer_info.rs:51]: do we need to skip the serialization? 
Enr has a `serde` feature which implements this. Might be nice to have.

Comment (realbigsean) [reply beacon_node/eth2_libp2p/src/peer_manager/peer_info.rs]: removed the skip 👍

File: beacon_node/http_api/tests/tests.rs [hunk 2]
```diff
@@ -1007,6 +1028,69 @@ impl ApiTester {
...
+        self
+    }
+
+    pub async fn test_get_node_peers(self) -> Self {
+        let result = self.client.get_node_peers().await.unwrap().data;
+
+        let expected = PeerData {
+            peer_id: self.external_peer_id.to_string(),
+            enr: None,
+            address: EXTERNAL_ADDR.to_string(),
```
Comment (AgeManning) [inline beacon_node/http_api/tests/tests.rs:1084]: I think this is about to be changed

**Review by AgeManning — COMMENTED** [review]
I think the endpoint name is about to change a little. See here: https://github.com/ethereum/eth2.0-APIs/pull/101

Also I think it expects us to return the peers in alphabetical order via peer_id

**Comment by realbigsean** [comment]
> I think the endpoint name is about to change a little. See here: [ethereum/eth2.0-APIs#101](https://github.com/ethereum/eth2.0-APIs/pull/101)
> 
> Also I think it expects us to return the peers in alphabetical order via peer_id

I left a comment there, cause the alphabetical ordering seems a little weird. Updated `address` to `last_seen_p2p_address` though.

**Review by AgeManning — APPROVED** [review]
Sounds good. Lets merge for now and keep an eye on the spec

**Pull request closed without merging**